src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -1411,10 +1435,13 @@ export class QuickInputController extends Disposable {
 							const items = input.items.slice();
 							const removed = items.splice(index, 1);
 							const activeItems = input.activeItems.filter((ai) => ai !== removed[0]);
+							const resetScrollPositionBefore = input.resetScrollPosition;
+							input.resetScrollPosition = false;
```

Comment:
It still resets when you remove the item?

Reply:
Yes because all it's doing is setting `items` which triggers an `update` which triggers the reset of the scroll position.